flink-end-to-end-tests/test-scripts/common_kubernetes.sh

```diff
@@ -75,7 +75,8 @@ function start_kubernetes_if_not_running {
         # here.
         # Similarly, the kubelets are marking themself as "low disk space",
         # causing Flink to avoid this node (again, failing the test)
-        sudo CHANGE_MINIKUBE_NONE_USER=true minikube start --vm-driver=none \
+        # Use fixed version v1.16.9 because fabric8 kubernetes-client could not work with higher version under jdk 8u252
+        sudo CHANGE_MINIKUBE_NONE_USER=true minikube start --kubernetes-version v1.16.9 --vm-driver=none \
```

Comment:
which k8s version is the e2e test usually using?

Reply:
Currently, we are using minikube `v1.8.2`, the default corresponding Kubernetes version is `v1.17` by default.

Reply:
Okay, I see. Thx!

Does it make sense to run the K8s tests with multiple K8s versions to better detect such issues in the future? We test against the latest version (1.18), oldest maintained (1.16)?

Reply:
This is a great idea. I will create a ticket to track this.